data-raw/england-icb-summary-metrics.R

```diff
@@ -11,6 +11,12 @@ icb <- boundaries_icb22 |>
   mutate(icb22_name = str_remove_all(icb22_name, "^NHS ")) |>
   mutate(icb22_name = str_remove_all(icb22_name, " Integrated Care Board$"))
 
+lookup_lsoa11_lsoa21_icb22 <-
+  lookup_lsoa11_sicbl22_icb22_ltla22 |>
+  select(lsoa11_code, icb22_code) |>
+  left_join(lookup_lsoa11_lsoa21_ltla22) |>
```

Comment:
Unfortunately, this approach isn't valid. I will leave a more detailed in comment at the end of the review.